crates/utils/src/settings/structs.rs

```diff
@@ -18,6 +18,8 @@ pub struct Settings {
   pub(crate) email: Option<EmailConfig>,
   pub(crate) setup: Option<SetupConfig>,
   pub(crate) additional_slurs: Option<String>,
+  pub(crate) community_name_max_length: Option<usize>,
+  pub(crate) user_name_max_length: Option<usize>,
```

Comment:
We should move some of these options into a new category so this doesnt get too long, but i'm not sure which ones or how to call it.

Reply:
Here we could instead have a single setting actor_name_max_length which applies to users and communities both.

Reply:
I agree that it should be a single setting, actor_name_length. And its only worth extricating into its own block, if they get a collection of their own settings.

Also keep in mind that the `site` table also has its own group of settings, I've opted for most of them to go in there rather than the config file. This one could go either way but its probably fine in the config.hjson.